webapp/src/components/SynthesisInformation.vue

```diff
@@ -1,20 +1,35 @@
 <template>
-  <div id="synthesis-information">
-    <label class="mr-2 pb-2">Synthesis Information</label>
-    <div class="card component-card">
-      <div class="card-body pt-2 pb-0 mb-0 pl-5">
-        <CompactConstituentTable
-          id="synthesis-table"
-          v-model="constituents"
-          :types-to-query="['samples', 'starting_materials']"
-        />
+  <div :id="block_id" class="data-block">
+    <div class="datablock-header collapsible" :class="{ expanded: isExpanded }">
```

Comment:
```suggestion
    <div class="datablock-header collapsible" :class="{ expanded: isExpanded }">
```

some of the labelling here is likely the issue too, make sure the css classes are appropriately named (see comment below)